client/src/ui/css/NotFound.module.css

```diff
@@ -0,0 +1,25 @@
+.notFoundMarginTop{
+  margin-top: 100px;
+}
+
+.notFoundImage {
+  width: auto;
+  height: 300px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 10px;
+}
+
+.notFound404Text {
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 50px;
+  font-weight: var(--regular-weight);
```

Comment:
great job converting this its own CSS Module! The annoying part containerizing CSS code is that you can't use global variables without importing files.

so, you need to add this at the top of the file (you probably won't see a visual change in the UI because we are just setting `font-weight` to a default value): 
```css
@import "../../index.css";
```